### PR TITLE
Remove unnecessary node, cluster and network setup

### LINKED ISSUE #13: Make testenv not suck (again)
`testenv` used to be extremely solid and reliable, however overtime it's become riddled with strange issues effecting ipfs, and ipfs cluster. Chances are this is due to the way `testenv` was designed long ago and that design hasn't really changed, despite massive improvements to IPFS and IPFS Cluster.

* cluster nodes not peering
* ipfs nodes dying
* excessive
* Issues spinning up on different environments (ie, it may work for setup X, but not setup Y) which makes developing on projects using `testenv` difficult

To improve we should:

* Reduce ipfs node count to 1
* Reduce ipfs cluster node count to 1 (ipfs cluster spins up single node clusters perfectly)
* Ideally get rid of the interface mocking via `ip addr` as this isn't guaranteed to behave properly across platforms. Because we will be using single nodes, we wont have to worry about port conflicts within the containers themselves.  In theory this should drastically simplify things since we don't have to deal with adding new low level ethernet interfaces
* Configuration files for ipfs and ipfs-cluster should either be completely regenerated, or modified in such a way that the chances of out-of-date config files is less

### PR DESCRIPTION
**Pull request by arku**
Fixes #13 

**Stuff done:**

- [x] Reduce `ipfs` node count to 1
- [x] Reduces `ipfs` cluster node count to 1
- [x]  Removes interface mocking via `ip addr` platforms.
- [ ] Regenerate configuration files for `ipfs` and `ipfs-cluster`

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -6,9 +6,9 @@ services:
     image: ipfs/go-ipfs:v0.4.20
```
Comment (bonedaddy) [inline docker-compose.yml]:
```suggestion
    image: ipfs/go-ipfs:v0.4.21
```

File: docker-compose.yml [hunk 2]
```diff
@@ -20,43 +20,12 @@ services:
       - ipfs.node.1
     image: ipfs/ipfs-cluster:v0.10.1
```
Comment (bonedaddy) [inline docker-compose.yml]:
```suggestion
    image: ipfs/ipfs-cluster:v0.11.0-rc3
```

File: ipfs/nodes/1/config [hunk 3]
```diff
@@ -9,7 +9,7 @@
   "Addresses": {
     "API": "/ip4/0.0.0.0/tcp/5001",
     "Announce": [
-      "/ip4/192.168.2.101/tcp/4001"
+      "/ip4/127.0.0.1/tcp/4001"
```
Comment (bonedaddy) [inline ipfs/nodes/1/config]:
```suggestion
      "/ip4/0.0.0.0/tcp/4001"
```

**Review by bonedaddy — CHANGES_REQUESTED** [review]
Looks really good, just a few small nit-picks i've suggested changes for, well done :muscle:

**Comment by arku** [comment]
@postables Done :slightly_smiling_face:

**Comment by bonedaddy** [comment]
Thanks! I'm going to try to run a complete Temporal API test locally and if that works then I'll approve :muscle:

**Comment by arku** [comment]
Sounds good, thanks!

**Review by bonedaddy — APPROVED** [review]
Awesome work, this will make development much easier

**Pull request merged**